README.md

```diff
@@ -52,6 +52,11 @@ Check the [releases](https://github.com/slackhq/nebula/releases/latest) page for
     $ brew install nebula
     ```
 
+- [Docker](https://hub.docker.com/r/nebulaoss/nebula)
+    ```
+    $ docker run nebulaoss/nebula
```

Comment:
```suggestion
    $ docker pull nebulaoss/nebula
```

Maybe? Run will definitely error without config and NET_ADMIN capabilities - and except for Lighthouses, without specifying a network flag, it isn't really useful to run anyway, I think.

Alternatively you could add a `#### Docker` section below and add the run command from the Docker Hub overview.

Reply:
I agree, lets do pull and they can go to the docker hub page for all the details